Source/Core/InputCommon/ControlReference/FunctionExpression.cpp

```diff
@@ -194,6 +207,292 @@ class WhileExpression : public FunctionExpression
   std::string GetFuncName() const override { return "while"; }
 };
 
+// usage: deadzone(input, amount)
```

Comment:
Is it really `deadzone` and not `!deadzone` (like the other expressions?)

Reply:
Nope. Fixed. :)